sites/website/src/docs/design-systems/high-contrast.md

```diff
@@ -0,0 +1,194 @@
+---
+id: high-contrast
+title: High Contrast in FAST
+sidebar_label: High Contrast
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/high-contrast.md
+---
+
+## Styling components using forced-colors.
+High contrast mode uses the CSS media feature, `forced-colors`. When `forced-colors` is set to active, the user agent will apply a limited color palette to the component.
```

Comment:
I think we should link externally to https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors here so folks are able to learn more about what forced-colors are if they're unfamiliar.